src/net.cpp

```diff
@@ -1003,24 +1005,684 @@ size_t V1Transport::GetSendMemoryUsage() const noexcept
     return m_message_to_send.GetMemoryUsage();
 }
 
+namespace {
+
+/** List of short messages as defined in BIP324, in order.
+ *
+ * Only message types that are actually implemented in this codebase need to be listed, as other
+ * messages get ignored anyway - whether we know how to decode them or not.
+ */
+const std::array<std::string, 33> V2_MESSAGE_IDS = {
+    "", // 12 bytes follow encoding the message type like in V1
+    NetMsgType::ADDR,
+    NetMsgType::BLOCK,
+    NetMsgType::BLOCKTXN,
+    NetMsgType::CMPCTBLOCK,
+    "", /* FEEFILTER is not implemented in Dash */
```

Comment:
"FEEFILTER_NOT_SUPPORTED" is better than empty string here IMO, to avoid mixing with other possible errors

```suggestion
    "FEEFILTER_NOT_SUPPORTED",
```